_twig/docker-compose.yml/service/console.yml.twig

```diff
@@ -16,9 +16,12 @@ services:
       - ./.my127ws/application:{{ @('app.code_owner_home') }}/application
       - ./.my127ws/docker/image/console/root/usr/lib/task:/usr/lib/task
       - ./.my127ws:/.my127ws
-{% else %}
```

Comment:
Intentional?

Reply:
nope, sorry initially I was thinking develop only then decided also for static, but then didn't notice the else wasn't and endif